src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/main.js

```diff
@@ -20,6 +20,14 @@ define([
     var defaults = {};
 
     return {
+
+        collaboration: function() {
+            return {
+                id: this.options.id,
+                type: 'collection'
+            };
+        },
```

Comment:
Do we really also want that in the entire collection? Could imagine that collections are often opened at the same time, would only be a problem for the overlays. I think I would just skip this component for now...